lib/fizzy/execute.cpp

```diff
@@ -598,13 +598,13 @@ execution_result execute(Instance& instance, FuncIdx func_idx, std::vector<uint6
             const auto target_imm = read<uint32_t>(immediates);
             LabelContext label{code.instructions.data() + target_pc,
                 code.immediates.data() + target_imm, arity, stack.size()};
-            labels.emplace_back(label);
+            labels.push(label);
             break;
         }
         case Instr::loop:
         {
             LabelContext label{pc - 1, immediates, 0, stack.size()};  // Target this instruction.
-            labels.push_back(label);
```

Comment:
Even currently this is inconsistent, in some place it uses emplace, in others push for labels.